AWSAppSyncClient/AWSAppSyncHTTPNetworkTransport.swift

```diff
@@ -6,6 +6,7 @@
 
 import Foundation
 import AWSCore
+import AppSyncRealTimeClient
```

Comment:
will be removing this `import` as it is unsed, was trying out the `AtomicValue` type that we have in AppSyncRealTimeClient that uses the NSLock earlier